frontend/src/app/cache.ts

```diff
@@ -78,6 +85,11 @@ export const selectFCEResultsForInstructor =
     (state: RootState): FCE[] | undefined =>
       state.cache.instructorResults[name];
 
+export const selectProfessors = (search: string) => (state: RootState) =>
+  state.cache.allProfessors.filter((prof) =>
+    prof.name.toLowerCase().includes(search.toLowerCase())
```

Comment:
i wonder if we can improve this logic.
for example, searching professors by (firstname lastname) instead of (lastname, firstname)

Reply:
could also be a feature for a later PR